docs/conf.py

```diff
@@ -0,0 +1,75 @@
+"""
+Configuration file for the Sphinx documentation builder.
+
+This file only contains a selection of the most common options. For a full
+list see the documentation:
+https://www.sphinx-doc.org/en/master/usage/configuration.html
+"""
+
+# -- Project information -----------------------------------------------------
+# import your own package here
+import fits_schema
+
+project = "fits_schema"
+#copyright = "VODF"
+author = "Maximilian Linhoff"
+version = fits_schema.__version__
+# The full version, including alpha/beta/rc tags.
+release = version
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx.ext.githubpages",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "numpydoc",
+    "sphinx_changelog",
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = []
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = ["changes"]
+
+# have all links automatically associated with the right domain.
+default_role = "py:obj"
+
+
+# intersphinx allows referencing other packages sphinx docs
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3/", None),
+    "astropy": ("https://docs.astropy.org/en/stable", None),
+    "numpy": ("https://numpy.org/doc/stable", None),
+
+}
+
+# -- Options for HTML output -------------------------------------------------
+
+html_theme = "ctao"
```

Comment:
Do you really want to use the ctao theme here? 

Maybe just go with the plain `pydata_sphinx_theme?`

Reply:
No, that was an artifact from copying the template. Changed to pydata.